lib/pages/main/wallet_contents/add_wallet_connect/add_wallet_connect.dart

```diff
@@ -254,27 +272,68 @@ class _AddWalletConnectState extends State<AddWalletConnect> {
     }
   }
 
+  String? validateWalletConnectURL(String? valueCandidate) {
+    final l10n = l10nOf(context);
```

Comment:
- wallet_connect_methods.dart in lib/helpers contains similar method. Please define only one

Reply:
It starts with "wc:", Resolved